sycl/source/accessor.cpp

```diff
@@ -13,8 +13,13 @@
 namespace sycl {
 inline namespace _V1 {
 namespace detail {
-device getDeviceFromHandler(handler &CommandGroupHandlerRef) {
-  return CommandGroupHandlerRef.MQueue->get_device();
+device getDeviceFromHandler(handler &cgh) {
+  assert((cgh.MQueue || cgh.MGraph) &&
+         "One of MQueue or MGraph should be nonnull!");
+  if (cgh.MQueue)
+    return cgh.MQueue->get_device();
+  else
+    return cgh.MGraph->getDevice();
```

Comment:
https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return
```suggestion
  if (cgh.MQueue)
    return cgh.MQueue->get_device();

  return cgh.MGraph->getDevice();
```